index.bs

```diff
@@ -145,8 +145,8 @@ class AV1CodecConfigurationBox extends FullBox('av1C', version = 0, 0){
 }
 
 aligned (8) class AV1CodecConfigurationRecord {
-  unsigned int (6) reserved = 0;
-  unsigned int (2) OBU_length_mode;
+  unsigned int (7) reserved = 0;
```

Comment:
Seems this stacked w/the OBU_LENGTH patch. Needs the 4 here.

Reply:
Yes, I must have messed up when I created the branch ... I will make sure the OBU_LENGTH commit is removed from this branch when I merge.